### DISCUSSION TITLE
Wine + LaserGRBL + /dev/ttyUSB0

### DISCUSSION THREAD

**Discussion by Pim-tech** [arkypita/LaserGRBL#1956] [General]
I, everybody.
I'm jeanyves from France,
I installed wine on my Linux debian 11 , then installed LaserGRBL latest binary,
then i parsed all documentation about configuring the ports :
made all stuff into regedit, made the symlinks into dosdevices directory.
but the ports are still unavalable. 
Should i need a patched version to have it work with wine?
I do not know a method to test the COM port into Wine : who knows?

But i fear the the problem is inside the program because it has been designed to work
with a windows driver and i'm pretty sure it makes a workaround instead of accessing my 
Linux ports.

My /dev/ttyUSB0 is accessible : i may cat messages from it.
And tried to send a small .nc file and  my machine did it.

So is there a patch or a specific version for my situation?

Another problem is that the program is very very slow with wine and we should say it blocks somewhere.
I could not see anything to debug unfortunatly. Am i wrong?

**Comment by Spartrap** [comment]
I have it working. My Wine is old, v3.0.2.
After version of wine 2.8, you only need to:
$ wine regedit
Open HKEY_LOCAL_MACHINE\Software\Wine\Ports
Create a new "String value". Name it "COM1". Edit that key's value and put in "/dev/ttyUSB0"
Close regedit. Close the wineserver:
$ wineserver -k
You are good to go..
